src/pybind/mgr/dashboard/frontend/e2e/cluster/osds.e2e-spec.ts

```diff
@@ -32,4 +33,51 @@ describe('OSDs page', () => {
       expect(osds.getTabText(1)).toEqual('Overall Performance');
     });
   });
+
+  describe('check existence of fields on OSD page', () => {
+    it('should check that number of rows and count in footer match', () => {
+      osds.navigateTo();
+      osds
+        .getTableCount()
+        .getText()
+        .then((tableCount) => {
+          tableCount = tableCount.slice(13, -6); // Grabs number of hosts from table footer
+          expect(osds.getTableRows().count()).toMatch(tableCount);
+        });
+    });
+
+    it('should verify that selected footer increases when an entry is clicked', () => {
+      osds.navigateTo();
+      osds.getFirstCell().click(); // clicks first osd
+      osds
+        .getTableCount()
+        .getText()
+        .then((tableCount) => {
+          tableCount = tableCount.substring(0, 1); // Grabs number of hosts from table footer
+          expect(tableCount).toMatch('1');
```

Comment:
This test can be done by a simple unit test.
You should open the different detail tabs and observe if they are filled to make a valid integration test.